src/lib/core/ripple/ripple.ts

```diff
@@ -13,10 +13,14 @@ import {
 import {RippleConfig, RippleRenderer} from './ripple-renderer';
 import {ViewportRuler} from '../overlay/position/viewport-ruler';
 import {RippleRef} from './ripple-ref';
+import {extendObject} from '../util/object-extend';
 
 /** OpaqueToken that can be used to globally disable all ripples. Except programmatic ones. */
 export const MD_DISABLE_RIPPLES = new OpaqueToken('md-disable-ripples');
 
+/** OpaqueToken that can be used to specify a global ripple configuration. */
+export const MD_RIPPLES_CONFIG = new OpaqueToken('md-ripples-config');
```

Comment:
I'm fine with that. Let me change that. Also you asked about the motivation of this change. 
* It's just really convenient if developers can globally modify the `speedFactor` for example.

See https://github.com/angular/material2/issues/2901#issuecomment-283648158

I'm personally also fine with only supporting `MD_RIPPLES_SPEEDFACTOR`, but I think that this feature might help us in the future as well.